tests/wrappers/test_record_episode_statistics.py

```diff
@@ -24,6 +26,17 @@ def test_record_episode_statistics(env_id, deque_size):
     assert len(env.length_queue) == deque_size
 
 
+def test_record_episode_statistics_reset_info():
+    env = gym.make("CartPole-v1")
+    env = RecordEpisodeStatistics(env)
+    obs = env.reset()
+    assert isinstance(obs, np.ndarray) or isinstance(obs, tuple) or isinstance(obs, int)
```

Comment:
So two points, applying to this specific line and the ones similar in other parts of the tests.
1. As mentioned in another comment, you can parametrize the test so that it runs for all environments, so that they're all checked and not just CartPole
2. I think a better to check for validity of the observation would be `env.observation_space.contains(obs)`. It generalizes to all other weirder action spaces (for example this version would fail with a Dict action space)

Reply:
1. I switched the direct for all first party environments in test_envs to the parameterization decorator. For now I'm leaving the wrapper tests as single-environment tests, since many existing tests are like this, and it might make the tests take much longer. If people think it's worthwhile, I'm happy to add more coverage such that we try every first party env with every wrapper/vector env.

2. I made this switch for all the direct first party env tests  wrapper tests except normalize.py because it seems to not contain the observation in the observation space! This might be an error worth investigating. I also didn't implement this for the vector env tests.